sw/device/tests/sim_dv/uart_tx_rx_test.c

```diff
@@ -0,0 +1,331 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/arch/device.h"
+#include "sw/device/lib/base/log.h"
+#include "sw/device/lib/base/mmio.h"
+#include "sw/device/lib/dif/dif_plic.h"
+#include "sw/device/lib/dif/dif_uart.h"
+#include "sw/device/lib/handler.h"
+#include "sw/device/lib/irq.h"
+#include "sw/device/lib/runtime/hart.h"
+#include "sw/device/lib/testing/check.h"
```

Comment:
This file doesn't exist. Should it be`sw/device/lib/runtime/check.h`?

Reply:
Yeah, I have to rebase and resubmit due to other changes.